src/caffe/proto/caffe.proto

```diff
@@ -1277,6 +1364,11 @@ message V1LayerParameter {
   optional V0LayerParameter layer = 1;
 }
 
+
+message L1LossParameter {
+  optional bool debug_info = 1 [default = false];
+}
```

Comment:
This can be removed - the debug_info prints from L1 loss layer has been removed. This layer doesn't need a parameter any more.